core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -597,7 +597,7 @@ object KafkaConfig {
   val LogRollTimeJitterMillisDoc = "The maximum jitter to subtract from logRollTimeMillis (in milliseconds). If not set, the value in " + LogRollTimeJitterHoursProp + " is used"
   val LogRollTimeJitterHoursDoc = "The maximum jitter to subtract from logRollTimeMillis (in hours), secondary to " + LogRollTimeJitterMillisProp + " property"
 
-  val LogRetentionTimeMillisDoc = "The number of milliseconds to keep a log file before deleting it (in milliseconds), If not set, the value in " + LogRetentionTimeMinutesProp + " is used"
+  val LogRetentionTimeMillisDoc = "The number of milliseconds to keep a log file before deleting it (in milliseconds), If not set, the value in " + LogRetentionTimeMinutesProp + " is used. If set to -1, no time limit is applied."
```

Comment:
Wouldn't "If set to a negative number, no time limit is applied." be a more accurate statement?

Reply:
Hi @vahidhashemian , technically you are indeed completely right, I simply copied over the existing sentence from the documentation, in order to keep them in line with each other.
I thought about changing it, but went with the same reasoning that we applied in a [similar discussion](https://github.com/apache/kafka/pull/6205#discussion_r254739039) we had a little while ago.

Reply:
Thanks for clarifying. I though we already had discussed this but couldn't quickly find it out :)
LGTM.